opus/application/apps/ui/templates/base.html

```diff
@@ -289,18 +289,31 @@ <h3 class='op-header-text'></h3>
             confirm_title="Confirm Reset Search"
             confirm_msg="This will delete any search constraints you have selected and reset the set of search fields back to the default. The selected metadata fields and the contents of your cart will not be affected. Are you sure you want to reset your search?"
             target="op-reset-search-modal"
+            yes_text="Yes"
+            no_text="No"
         %}
         {% include "confirm_modal.html" with
             id_name="op-reset-search-metadata-modal"
             confirm_title="Confirm Reset Search and Metadata"
             confirm_msg="This will delete any search constraints you have selected and reset the set of search fields and selected metadata back to the default. The contents of your cart will not be affected. Are you sure you want to reset your search and metadata?"
             target="op-reset-search-metadata-modal"
+            yes_text="Yes"
+            no_text="No"
         %}
         {% include "confirm_modal.html" with
             id_name="op-empty-cart"
             confirm_title="Confirm Remove All Observations From Cart"
             confirm_msg="Are you sure you want to remove all observations from your cart?"
             target="op-empty-cart"
+            yes_text="Yes"
+            no_text="No"
+        %}
+        {% include "confirm_modal.html" with
+            id_name="op-cart-status-error-msg"
+            confirm_title="Message About Cart Status"
```

Comment:
It's only possible to have a failure return with a status code on add/addrange/addall, so let's make the title something more meaningful to the user: "Failed to Add Observations to Cart".